gr-uhd/lib/usrp_source_impl.h

```diff
@@ -113,7 +113,7 @@ class usrp_source_impl : public usrp_source, public usrp_block_impl
     //! Like set_center_freq(), but uses _curr_freq and _curr_lo_offset
     ::uhd::tune_result_t _set_center_freq_from_internals(size_t chan,
                                                          pmt::pmt_t direction) override;
-    void _cmd_handler_tag(const pmt::pmt_t& tag);
+    void _cmd_handler_tag(const pmt::pmt_t& tag, int chan, const pmt::pmt_t& msg);
```

Comment:
```suggestion
    void _cmd_handler_tag(const pmt::pmt_t& tag);
```

Reply:
Done, the latest push keep signature of ``_cmd_handler_tag()`` unchanged